### PR TITLE
tweaks on rewards

### PR DESCRIPTION
**Pull request by jenn-rhim**
- ~header top padding~
- darker description
- ~toggle~

### CODE REVIEW

**Comment by petemill** [comment]
I’d be happy to help clean up tweak PRs like this through review

**Comment by jenn-rhim** [comment]
@petemill that would be helpful!

**Comment by petemill** [comment]
@jenn-rhim I've got a different solution that I'd prefer because it cleans this up a bit and simplifies the code: It essentially just says here's the graphic, title and toggle and asks the browser to "please vertically center them" :-) Then it can use your explicit consistent top and bottom padding of 18px.

This also allows the 'off' version to keep that padding instead of inherit the padding values that were only there to push the 'on' content to the center.

This PR can then remain to change the rewards box text color default.

We still need confirmation that the Toggle needs to change on every Layout we have. Perhaps that can be explained in a different medium, i.e. does the current Toggle implementation width and border-radius differ from the Sketch library? We can certainly fix it.

@jenn-rhim can you take a look at the below and confirm:
- [ ] Is the vertical alignment 👍 
- [ ] Which TM position is best, or any changes on my offering?
- [ ] In my version, is the logo vertical centering bothering you? If so, the actual graphic needs to change so that the desired center point of the graphic is in the middle.

## Before
image


## With this PR
image


## With simplified vertical centering PR
_Consistent 18px top and bottom padding with rest of height defined by tallest item (bat logo told to have 66px height)_

image

_note that the logo **is** actually centered_
image

**Comment by jenn-rhim** [comment]
I think this is good and a better solution. I didn't touch the logo because the storybook structure didn't make it clear how and where I can do that in my Visual Studio. Wanted to do it at a later time and actually thought I'd just express the design intent of centering the logo. So, thank you, you killed both birds!

File: src/features/rewards/box/style.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const StyledDescription = styled<{}, 'div'>('div')`
   font-size: 14px;
   line-height: 1.29;
   letter-spacing: normal;
-  color: #a4aeb8;
+  color: #838391;
```
[the remark this answers is not in the document]
Comment (jenn-rhim) [reply src/features/rewards/box/style.ts]: this is one of the grays in our palette, i believe @pete.

File: src/components/formControls/toggle/style.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const StyledSlider = styled<Props, 'div'>('div')`
   height: ${(p) => p.size === 'small' ? '6px' : '8px'};
   margin-top: ${(p) => p.size === 'small' ? '5px' : '8px'};
   width: 100%;
-  border-radius: 3px;
```
[the remark this answers is not in the document]
Comment (jenn-rhim) [reply src/components/formControls/toggle/style.ts]: ok. @pete. we can wait.

File: src/features/rewards/box/style.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pete) [reply src/features/rewards/box/style.ts]: Thank you, @jenn-rhim.  It is a very nice grey.

File: src/components/formControls/toggle/style.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pete) [reply src/components/formControls/toggle/style.ts]: There's no need to wait on my account.

**Pull request merged**